airwallex/src/main/java/com/airwallex/android/view/PaymentMethodsActivity.kt

```diff
@@ -181,11 +191,15 @@ class PaymentMethodsActivity : AirwallexCheckoutBaseActivity() {
 
         if (session is AirwallexPaymentSession) {
```

Comment:
Beyond the scope of this PR obviously, but this stuff needs to be moved into the view model so it can be tested

Reply:
I agree, will add a TODO: and extract it later when we have time